### DISCUSSION TITLE
Recover Harvard CS50AI plyus python coding

### DISCUSSION THREAD

**Discussion by MyHarvardCode** [community/community#147629] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Please help. I lost password for Github after short break from coding. Created a new password and that action erased all my coding for Harvard CS50 AI plus python. Is it possible to recover my coding? Many thanks, Dr Ralph

**Comment by shinybrightstar** [comment]
Welcome to the GitHub Community, @MyHarvardCode , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. This belongs to the General Category, I've gone ahead and moved it for you. Good luck!

**Reply by MyHarvardCode** [reply]
Many thanks for responding. I don't want to lose this message before going to General Category. How do I get there?

**Comment by MyHarvardCode** [comment]
Many thanks for your reply.
Please tell me how to go to the Github general category.

All the Best, Ralph



Dr. Ralph J. Lamson
PO Box 6891
San Rafael, California 94903-0891

Best Books - Here is the Link:
*https://shepherd.com/best-books/self-realization-curiosity-exploration-struggle*
<https://shepherd.com/best-books/self-realization-curiosity-exploration-struggle>

courses_completed = { *'Stanford*': ['CIP'], '*DeepLearning.AI*': ['AI
Python', 'Coding Automating Tasks with Python', 'LLM Prompting with
Gemini', 'Pretraining LLMs', 'ChatGPT', 'lamini' LLM] '*Harvard*': ['AI
Python']}

*Trauma-Informed Interviewing Certified*
*Office of Democratic Institutions and Human Rights*
*OSCE - ODIHR*

*LinkedIn Leadership Think Tank*

SPIT-C Stanford Psychiatry Immersion Technology - Consortium, 2019-2023

*Author*: *Virtual Therapy: prevention and treatment of psychiatric
conditions by immersion in virtual reality environments, Polytechnic
International Press, University of Montreal, Canada*

*Inventor*,* Virtual Reality Immersion Therapy (VRIT)*
US Patent 6425764



*May truth rise above all that wants to diminish it. **rjl*


"Notice: No duties are assumed, intended or created by this
communication. You are encouraged to retain counsel of your choice if you
desire to do so."
